Service/ContentTypeService.php

```diff
@@ -356,6 +356,16 @@ public function getByName($name)
         return $this->contentTypeArrayByName[$name] ?? false;
     }
 
+    public function getByNames(array $names) : array
+    {
+        $this->loadEnvironment();
+        $contentTypes = [];
+        foreach ($names as $name) {
+            $contentTypes[] = $this->contentTypeArrayByName[$name] ?? null;
+        }
+        return $contentTypes;
+    }
+
```

Comment:
This function adds 10 lines of code, while the calling code can handle this in 1 line of code. This function is not needed.

```suggestion
```

Reply:
9e53b2e7dded6dba8a74028e301873ec791a5c71